.github/workflows/integration_tests.yml

```diff
@@ -24,6 +24,10 @@ on:
         description: 'CSV of desktop SSL variants to use'
         default: 'openssl,boringssl'
         required: true
+      mobile_test_on:
+        description: 'Mobile integration_test apps test on FTL or iOS Simulator/Android Emulator'
```

Comment:
nit: this could be phrased a little shorter: "Run mobile tests on real and/or virtual devices? (separated by commas)"